README.md

```diff
@@ -0,0 +1,19 @@
+# Release Information
+
+* **Version**: 1.0.0
+* **Certified**: No
+* **Publisher**: Fortinet
+* **Compatible Version**: FortiSOAR v7.2.2
```

Comment:
Why 7.2.2 only?

Reply:
info.json has value --> "fsrMinCompatibility": "7.2.0",

Reply:
The SOC management widget has a minimum compatibility of 7.2.2